.travis.yml

```diff
@@ -32,12 +39,18 @@ install:
     - pip install --no-index --find-links=travis_wheels -r requirements/compiled.txt
     - pip install --find-links=base_wheels -r requirements/prod.txt -r requirements/dev.txt
     - pip install coveralls
+    # JS test requirements
+    - curl -sL https://deb.nodesource.com/setup | sudo bash -
+    - sudo apt-get install -y nodejs
 
 before_script:
     - mysql -e 'create database kuma;'
 
 script:
     - coverage run --include=`pwd`/kuma/*,`pwd`/apps/* manage.py test --noinput -v2 access actioncounters authkeys contentflagging devmo kpi kuma landing sumo
+    - cd tests/ui
+    - npm install intern
```

Comment:
the npm install should not be part of the testing scripts but of the setup (probably ``install`` in that case). A failure during one of the scripts does not stop Travis from running other ones (this is a main difference with other steps of the build)